src/main/target/BETAFLIGHTF3/target.c

```diff
@@ -29,19 +29,18 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM4,  CH2,  PB7, TIM_USE_PPM,                 0), // PPM  DMA(1,4)
 
     // Motors 1-4
-    DEF_TIM(TIM16, CH1,  PA6, TIM_USE_MOTOR,               0), // PWM1 DMA(1,6)
-    DEF_TIM(TIM8,  CH1N, PA7, TIM_USE_MOTOR,               0), // PWM2 DMA(2,3)
-    DEF_TIM(TIM8,  CH2,  PB8, TIM_USE_MOTOR,               0), // PWM3 DMA(2,5)
-    DEF_TIM(TIM17, CH1,  PB9, TIM_USE_MOTOR,               0), // PWM4 DMA(1,7)
+    DEF_TIM(TIM16, CH1,  PA6, TIM_USE_MOTOR,               0), // PWM1 UP(1,6)
```

Comment:
This will still build with DMA instead of DMAR:

https://github.com/betaflight/betaflight/pull/4953/files#diff-9fd9f2d38905862b32820d5ba411ee29R33

If DMAR works, this should be changed to `#define USE_DSHOT_DMAR`.

Reply:
Fixed.